lib/files.py

```diff
@@ -91,8 +91,12 @@ def read_zulip_messages_for_topic(
     f.close()
     return messages
 
-def open_config(md_root):
-    outfile = open_outfile(md_root, Path('_config.yml'), 'w+')
+def open_read_config(md_root):
+    outfile = open_outfile(('..' / Path(md_root)).resolve(), Path('_config.yml'), 'r+')
```

Comment:
This is not general -- it assumes that md_root is `./archive`. Some users might choose to put it in a different path altogether, and the `..` of that path is not the root of the repo. Although one solution is to force the archive directory to always be located in the root of the repo. Since style.css is also located in the repo root. Thoughts @showell ?